specifyweb/frontend/js_src/lib/localization/locality.ts

```diff
@@ -292,4 +292,42 @@ export const localityText = createDictionary({
     'uk-ua': 'DD MM SS.SS N/S/E/W (32 45 42.84 N)',
     'de-ch': 'DD MM SS.SS N/S/O/W (32 45 42.84 N)',
   },
+  localityImportHeaderError: {
+    'en-us': 'Errors Found in Column Headers',
+  },
+  localityImportMissingHeader: {
+    'en-us': 'The following columns are required but missing in the dataset',
+  },
+  localityImportUnrecognizedHeaders: {
+    'en-us':
+      'The following columns in the dataset are not recognized and will be ignored on import',
+  },
+  localityImportedAcceptedHeaders: {
+    'en-us': 'Only the following headers are accepted',
+  },
+  localityImportErrorDialogHeader: {
+    'en-us': 'Error(s) Occured while Parsing Dataset',
```

Comment:
we normally don't use Header Case in dialog headers
maybe we should, but for concistency we should either do so everywhere, or nowhere